README.md

```diff
@@ -93,14 +93,14 @@ _Note:_ If you don't see this checkbox, make sure you're logged in to Airtable,
 Once you have your API key, create a file called `.env` and add your API key:
 
 ```bash
-AIRTABLE_API_KEY=YOUR_API_KEY
+GATSBY_AIRTABLE_API_KEY=YOUR_API_KEY
 ```
 
 > **NOTE:** See `.env.EXAMPLE` for a copy-pasteable template to get set up!
 
 ### Airtable Base ID
 
-Additionally, the `.env.EXAMPLE` file has the `AIRTABLE_BASE_ID` pre-defined for you. This ID references the dummy database mentioned above. Copy and paste that directly into your newly created `.env` file as well.
+Additionally, the `.env.EXAMPLE` file has the `GATSBY_AIRTABLE_BASE_ID` pre-defined for you. This ID references the dummy database mentioned above. Copy and paste that directly into your newly created `.env` file as well.
```

Comment:
Looking over this again @SeHarlan - I'm wondering if we need to update the documentation to explain that the AIRTABLE_API_KEY and the AIRTABLE_BASE_ID need to be added to _both_ the `.env` and the `.env.development` files.

Because the Allies page, is directly querying the Airtable database, we still need to leverage our current implementation of `gatsby-source-airtable` which is using the `.env` variables on the server side to connect to Airtable.

Since we are _also_ now leveraging the Airtable API base and key on the client side, the environment variables that you have set up (`GATSBY_AIRTABLE_API_KEY` and `GATSBY_AIRTABLE_BASE_ID`) will need to be added to the `.env.development` file in addition to the environment variables mentioned above in the `.env` file.

This is definitely convoluted and a bit confusing at the moment. But this is a bit of tech debt that we have inherited from our initial launch, and will be cleaned up ASAP.

Happy to chat further about this, but the tl;dr of this message is to essentially just add another section to the README here explaining that the `GATSBY_` prefixed environment variables need to be added to `.env.development` in addition to the `.env.` file.

Reply:
@SeHarlan I'll actually take this on in a separate PR. We need to update our documentation round Airtable in general.

I think with that, @racedale was able to get the tests passing, and I think we can get this sucker merged!